### ISSUE TITLE
empty array and { default_flow_style: false } cause yaml.dump to throw an error

### ISSUE THREAD

**Issue by spect88** [connec/yaml-js#15]
Hi,

Running the following code
```javascript
yaml.dump({ empty: [] }, null, null, { default_flow_style: false })
```
results in `EmitterError: expected NodeEvent, but got SequenceEndEvent` being thrown.

PyYAML's output:
```python
>>> yaml.dump({ 'empty': [] }, default_flow_style=False)
'empty: []\n'
```

Am I doing something wrong here?

**Comment by connec** [comment]
There is probably a bug in the parser, I'll have a look asap and see what I can find.

**Comment by connec** [comment]
OK, that was a silly guess. The bug was a fairly ridiculous typo in the emitter.

I've fixed it and published 0.1.2.
